CMakeLists.txt

```diff
@@ -3,10 +3,14 @@ set(CMAKE_CXX_STANDARD 17)
 
 set(LOCAL_CFG "" CACHE FILEPATH "Path to local config")
 
-if(NOT LOCAL_CFG)
-    message(FATAL_ERROR "LOCAL_CFG variable not set. Use -DLOCAL_CFG=<file> to set local config. See local-example.cmake for available options")
+# if(NOT LOCAL_CFG)
+#     message(FATAL_ERROR "LOCAL_CFG variable not set. Use -DLOCAL_CFG=<file> to set local config. See local-example.cmake for available options")
+# endif()
```

Comment:
it would be good to not have commented out code :)

Reply:
Done